tests/test_monitor.py

```diff
@@ -34,3 +35,47 @@ def test_monitor():
     assert set(last_logline.keys()) == {'l', 't', 'r'}, "Incorrect keys in monitor logline"
     file_handler.close()
     os.remove(mon_file)
+
+def test_monitor_load_results(tmp_path):
+    """
+    test load_results on log files produced by the monitor wrapper
+    """
+    env1 = gym.make("CartPole-v1")
+    env1.seed(0)
+    monitor_file1 = os.path.join(tmp_path, "stable_baselines-test-{}.monitor.csv".format(uuid.uuid4()))
+    monitor_env1 = Monitor(env1, monitor_file1)
+
+    monitor_files = get_monitor_files(tmp_path)
+    assert len(monitor_files) == 1
+    assert monitor_file1 in monitor_files
+
+    monitor_env1.reset()
+    for _ in range(1000):
+        _, _, done, _ = monitor_env1.step(monitor_env1.action_space.sample())
+        if done:
+            monitor_env1.reset()
+
+    results_size1 = load_results(os.path.join(tmp_path)).size
+    assert results_size1 > 0
```

Comment:
you could predict that size by counting the number of episodes, no?

Reply:
See above. The numbers differ between test runs.

Reply:
Should I go back to using fixed action of 0 instead of sampling to get more predictable test runs?

Reply:
>The numbers differ between test runs.

that's normal but if you count the number of episode (looking the done signal), you can normally know in advance the size of the monitor file (cf evaluation helper: https://stable-baselines.readthedocs.io/en/master/_modules/stable_baselines/common/evaluation.html#evaluate_policy)

Reply:
Missed your point with counting episodes at first. Now implemented.